lib/gigya_flutter_plugin.dart

```diff
@@ -48,7 +48,8 @@ typedef void OnScreenSetEvent(event, data);
 /// sure you are using the same instance across your application.
 /// Using the singleton pattern here is crucial to prevent channels overlapping.
 class GigyaSdk with DataMixin {
```

Comment:
We approve adding null-safety for all methods but we do not approve switching all methods to " shorthand syntax".

Reply:
Changed